### PR TITLE
Fix #1256: Rework member profile page into two sections.

### LINKED ISSUE #1256: Rework Member Profile page into two sections
Rework Member Profile Page so that it is two columns instead of one.  Since left and the right sections have separate actions, I think it will be more intuitive this way.

Quick mockup for general idea:

two columns

### PR DESCRIPTION
**Pull request by khantaalaman**
### Proposed changes in this pull request

Fix #1256 
- Reworked member profile page into two sections.

### When should this PR be merged

[Please describe any preconditions that need to be addressed before we
can merge this pull request.]

No preconditions.

### Risks

[List any risks that could arise from merging your pull request.]

- NONE

### Follow up actions

[List any possible follow-up actions here; for instance, testing data
migrations, software that we need to install on staging and production
environments.]

- NONE

#### General

- [ ] **Is this PR explained thoroughly?** All code changes must be accounted for in the PR description. 
- [ ] **Is the PR labeled correctly?** It should have the `migration` label if a new migration is added. 
- [ ] **Is the risk level assessment sufficient?** The risks section should contain all risks that might be introduced with the PR and which actions we need to take to mitigate these risks. Possible risks are database migrations, new libraries that need to be installed or changes to deployment scripts.

#### Functionality

- [ ] **Are all requirements met?** Compare implemented functionality with the requirements specification.
- [ ] **Does the UI work as expected?** There should be no Javascript errors in the console; all resources should load. There should be no unexpected errors. Deliberately try to break the feature to find out if there are corner cases that are not handled. 

#### Code

- [ ] **Do you fully understand the introduced changes to the code?** If not ask for clarification, it might uncover ways to solve a problem in a more elegant and efficient way.
- [ ] **Does the PR introduce any inefficient database requests?** Use the debug server to check for duplicate requests. 
- [ ] **Are all necessary strings marked for translation?** All strings that are exposed to users via the UI must be [marked for translation](https://docs.djangoproject.com/en/1.10/topics/i18n/translation/). 

#### Tests

- [ ] **Are there sufficient test cases?** Ensure that all components are tested individually; models, forms, and serializers should be tested in isolation even if a test for a view covers these components. 
- [ ] **If this is a bug fix, are tests for the issue in place**  There must be a test case for the bug to ensure the issue won’t regress. Make sure that the tests break without the new code to fix the issue.
- [ ] **If this is a new feature or a significant change to an existing feature** has the manual testing spreadsheet been updated with instructions for manual testing?

#### Documentation

- [ ] **Are changes to the UI documented in the platform docs?** If this PR introduces new platform site functionality or changes existing ones, the changes must be documented in the [Cadasta Platform Documentation](https://github.com/Cadasta/cadasta-docs).
- [ ] **Are changes to the API documented in the API docs?** If this PR introduces new API functionality or changes existing ones, the changes must be documented in the [API docs](https://github.com/Cadasta/api-docs).
- [ ] **Are reusable components documented?** If this PR introduces components that are relevant to other developers (for instance a mixin for a view or a generic form) they should be documented in the Wiki.

### CODE REVIEW

**Comment by khantaalaman** [comment]
Supporting screenshot:

screenshot from 2017-03-09 21-38-50

Thanks!

**Comment by khantaalaman** [comment]
Hey @clash99, regarding suggested changes:

1. I did not make any change to `main.css`. Its surprising to see it getting into my PR. I'll make sure it removed. No worries.

2. Even I wanted `left-column` to be aligned with `title` and finally decided to make it exactly as it was in the mockup posted with the issue. I'll have the alignment adjusted as asked. 

3. This point is a bit confusing. The border is needed for `left-column` or the right one? Because before making the change, a border seems to be present in the right-column (https://cloud.githubusercontent.com/assets/1119621/23757299/063ae7d4-04ac-11e7-9078-02a62bf21884.png). 

I'll have things updated very soon.

Thanks for a prompt review! :)

**Comment by clash99** [comment]
Hi @khantaalaman - 

2) The mock wasn't exact so totally get it - sorry for the confusion.  Left aligned under the title is ideal.

3) Let's revert back to the default panel style (with a border) on the left.  This should match the right which I believe is the default too.

I hope that helps!  Thanks again, Chandra

**Comment by khantaalaman** [comment]
@clash99 , the PR has been updated with squashed commits.

Here's the latest screenshot. It looks good to me now. Please let me know about further improvements possible, if any:

screenshot from 2017-03-10 01-17-57

Thanks!

File: cadasta/templates/organization/organization_members_edit.html [hunk 1]
```diff
@@ -16,85 +16,81 @@
...
+              {% if not org_admin %}
+              <div class="panel-footer panel-buttons text-center">
+                <button type="submit" class="btn btn-primary" name="submit">{% trans "Save" %}</button>
+                <a href="{% url 'organization:members' organization.slug %}" class="btn btn-default">{% trans "Cancel" %}</a>
+              </div>
+              {% endif %}
+            </div>
+          </div>
+        </div>
+      </form>
```
Comment (clash99) [inline cadasta/templates/organization/organization_members_edit.html]: Div tags seem improperly nested at-a-glance.  I see two removed from the beginning but only from the bottom?  It should be one row with a col-md-3 and col-md-9.  Can you just double-check to make sure they are nested correctly with correct indentation, etc?  Thanks!

Comment (khantaalaman) [inline cadasta/templates/organization/organization_members_edit.html]: I had a cursory look to this and found that there is some problem with nesting of divs. This error was there from even before making changes to this file. No worries, I'll make necessary modifications required for correct nesting. 
PS: I'll submit a PR after some 12 hours as I have 2 mid semester exams scheduled today from after 4 hours from now. 

Thanks!

Comment (clash99) [inline cadasta/templates/organization/organization_members_edit.html]: No problem - good luck on your exams @khantaalaman!!

**Comment by khantaalaman** [comment]
@clash99 Please review this whenever you're comfortable. I believe to have resolved the minor error related to some divs being off. Sorry for the delay with this.

Thanks! :)

**Comment by khantaalaman** [comment]
@clash99 I have updated the PR. Please check if it is properly rebased. 

Thanks! :)

**Pull request merged**